bcda/client/bluebutton.go

```diff
@@ -103,10 +103,16 @@ func (bbc *BlueButtonClient) GetPatient(patientID, jobID, cmsID string) (string,
 	return bbc.getData(blueButtonBasePath+"/Patient/", params, jobID, cmsID)
 }
 
-func (bbc *BlueButtonClient) GetPatientByHICNHash(hashedHICN string) (string, error) {
+func (bbc *BlueButtonClient) GetPatientByIdentifierHash(hashedIdentifier, patientIdMode string) (string, error) {
 	params := GetDefaultParams()
+
+	identifier := "hicnHash"
```

Comment:
This should change to `hicn-hash` according to: https://github.com/CMSgov/beneficiary-fhir-data/blob/0877232340983fda8dbcd202012e991b8b531ac5/apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerConstants.java#L199

Reply:
so should this be hicnHash or hicn-hash. The link above says hicn-hash but in slack(https://adhoc.slack.com/archives/CK6Q7BBPC/p1580850698011000)  it says hicnHash ??

Reply:
Yeah, it should be `hicn-hash`.  It will work with `hicnHash` too, but if you look at the `TransformerConstants` file you'll see that things get complicated for us if we choose to continue to use `hicnHash`.
https://github.com/CMSgov/beneficiary-fhir-data/blob/0877232340983fda8dbcd202012e991b8b531ac5/apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerConstants.java